traitsui/wx/range_editor.py

```diff
@@ -201,25 +199,22 @@ def update_object_on_scroll(self, event):
              (event_type == wx.wxEVT_SCROLL_THUMBRELEASE))):
             try:
                 self.ui_changing = True
-                self.control.text.SetValue(self.format % value)
+                self.control.text.SetValue( self.format % value )
                 self.value = value
             except TraitError:
                 pass
             finally:
                 self.ui_changing = False
 
-    #-------------------------------------------------------------------------
+    #---------------------------------------------------------------------------
     #  Handle the user pressing the 'Enter' key in the edit control:
-    #-------------------------------------------------------------------------
+    #---------------------------------------------------------------------------
 
-    def update_object_on_enter(self, event):
+    def update_object_on_enter ( self, event ):
         """ Handles the user pressing the Enter key in the text field.
         """
-        if isinstance(event, wx.FocusEvent):
-            event.Skip()
```

Comment:
Behavioural change: This block is removed.

I think this block is needed though.